cli/cmd/compliance_aws.go

```diff
@@ -39,46 +39,14 @@ var (
 		Long:    `List all AWS accounts configured in your account.`,
 		Args:    cobra.NoArgs,
 		RunE: func(_ *cobra.Command, _ []string) error {
+			cli.StartProgress(" Fetching compliance information...")
```

Comment:
```suggestion
			cli.StartProgress(" Fetching list of configure AWS accounts...")
```

Reply:
Since you two asked to change this progress, will change it for the three clouds.  💯

Reply:
shouldn't the message say "Fetching list of **configured** AWS accounts"